README.md

```diff
@@ -7,6 +7,49 @@ height="42" />
 
 Next-generation WordPress eCommerce Experience for WordPress sites at Newfold Digital.
 
+
+## Module Responsibilities
+
+* Ecommerce module displays `Home or Dashboard` page for Brand plugin, with below options:
+  1. Live site preview for both, live and coming soon mode for WordPress site with view & edit site options
```

Comment:
This shouldn't be a concern of the e-commerce module, and we should strive to move them elsewhere.

Reply:
Created a `issue` for the same, please review once.
https://github.com/newfold-labs/wp-module-coming-soon/issues/35

Reply:
Created a GitHub issue for this change,  please review once
https://github.com/newfold-labs/wp-module-ecommerce/issues/283

Reply:
Please review, [JIRA](https://jira.newfold.com/browse/PRESS0-1251)